dist/src/collections.js

```diff
@@ -156,49 +164,62 @@ var addCollectionsApi = function addCollectionsApi(matroid) {
   };
 
   // https://www.matroid.com/docs/api/index.html#api-Collections-PostApiCollectionTasksTaskidImageQuery
-  matroid.queryCollectionByImage = function (taskId, taskType, numResults, image) {
+  matroid.queryCollectionByImage = function (taskId, image) {
     var _this9 = this;
 
-    var configs = arguments.length > 4 && arguments[4] !== undefined ? arguments[4] : {};
+    var configs = arguments.length > 2 && arguments[2] !== undefined ? arguments[2] : {};
 
     /*
-    Query against a collection index (CollectionManagerTask) using an image as key
+    Queries against a collection index using an image.
     */
     return new Promise(function (resolve, reject) {
-      _this9._checkRequiredParams({ taskId: taskId, taskType: taskType, numResults: numResults, image: image });
+      _this9._checkRequiredParams({ taskId: taskId, image: image });
 
       _this9._validateImageObj(image);
       _this9._checkImageSize(image.file);
 
       var options = {
         action: 'queryCollectionByImage',
         uriParams: { ':key': taskId },
-        data: { taskType: taskType, numResults: numResults }
+        data: {}
       };
 
-      if (image.file) options.filePaths = image.file;
-      if (image.url) Object.assign(options.data, { url: image.url });
-
-      var processedConfigs = configs;
-      var bbox = processedConfigs.boundingBox;
-      if (bbox) processedConfigs.boundingBox = JSON.stringify(bbox);
-
-      Object.assign(options.data, processedConfigs);
+      if (image.file) {
+        options.filePaths = image.file;
+      }
+      if (image.url) {
+        Object.assign(options.data, { url: image.url });
+      }
+
+      var numResults = configs.numResults,
```

Comment:
Maybe not for this PR, but I think we're good to move from `var` to `let / const` since we have a build process now.